File: app/src/main/java/br/com/victorlsn/mytaxi/interfaces/BaseMVP.java
package br.com.victorlsn.mytaxi.interfaces;

/**
 * Created by victorlsn on 26/02/19.
 */

public interface BaseMVP {

    interface Presenter {
        void attachView(View view);
    }

    interface View {
        void showProgressBar(boolean show, String message);

        void showToast(String message, int duration);
    }
}

File: app/src/main/java/br/com/victorlsn/mytaxi/interfaces/BaseMVP.kt
package br.com.victorlsn.mytaxi.interfaces

/**
 * Created by victorlsn on 26/02/19.
 *
 */

interface BaseMVP {

    interface Presenter {
        fun attachView(view: View)
    }

    interface View {
        fun showProgressBar(show: Boolean, message: String?)

        fun showToast(message: String, duration: Int)
    }
}